server/test/jobs/DurableJobRegistryTest.java

```diff
@@ -0,0 +1,56 @@
+package jobs;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import models.PersistedDurableJob;
+import org.junit.Test;
+
+public class DurableJobRegistryTest {
+
+  @Test
+  public void registeringAndAccessingAJob() throws Exception {
```

Comment:
Style: All the method names need to be be verb phrases

registerAndGetAJob

Reply:
Done